assets/dev/js/editor/views/add-section/base.js

```diff
@@ -6,12 +6,24 @@ import environment from 'elementor-common/utils/environment';
  */
  class AddSectionBase extends Marionette.ItemView {
 	static IS_CONTAINER_ACTIVE = ! ! elementorCommon.config.experimentalFeatures.container;
```

Comment:
Maybe '!!' instead of '! !'? 
@louiswol94

Reply:
@hein-obox There is a difference between `! !` and `!!`
This variable `elementorCommon.config.experimentalFeatures.container` will be `true` (when the experiment is active) or `undefined` (when the experiment is inactive).
The current expression using `! !` (with the space in between) will force it to return a `boolean` value true/false.